### ISSUE TITLE
strange toolbar with print button

### ISSUE THREAD

**Issue by scummos** [afrimberger/okularplugin#9]
The current version of the plugin displays two toolbars with two print buttons, one with and one without text; like this: http://i.imgur.com/bvdSO1P.png which is quite odd.
They can be removed, but re-appear when the plugin loads a different file. It'd be nice if the plugin would remember my choice not to show the toolbars.
On a similar topic, the main toolbar is now gone -- is there a reason for that, or is it a bug I experience only here?

**Comment by rubenvb** [comment]
Additionally, there is no way of saving the pdf file once it's loaded. This is quite a usability limitation unfortunately... If the kpartsplugin wasn't crashing, I'd use that over this.

**Comment by scummos** [comment]
Yeah, kpartsplugin is very nice too but it has its own share of problems ... esp. regarding focus and such.
Still, I think it's the better option unless this one gets back its toolbar :)
